responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -18,10 +18,8 @@ RUN pip install --pre 'azure-ai-ml' 'azure-storage-blob<=12.13.0' 'numpy<1.24.0'
 RUN pip install 'azureml-dataset-runtime=={{latest-pypi-version}}' \
                 'azureml-core=={{latest-pypi-version}}' \
                 'azureml-mlflow=={{latest-pypi-version}}' \
-                'azureml-rai-utils==0.0.2'
-
-# To resolve vulnerability issue
-RUN pip install 'Werkzeug==2.2.3'                
+                'azureml-rai-utils==0.0.2' \
+                'Werkzeug==2.2.3'
```

Comment:
Thanks for adding it above. It was in separate line because of the comment - Install azureml packages - may be we can remove comment or update it to accommodate Werkzeug

Reply:
this is just a copy of responsibleai repo - I will make this change there and then propagate here